.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file added to git if the .idea folder is gitignored?

Reply:
interesting ... will try to remove